<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDoseComparisonModule</class>
 <widget class="qSlicerWidget" name="qSlicerDoseComparisonModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoseComparison</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Parameter set:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ParameterSet">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLDoseComparisonNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton_Input">
     <property name="text">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Reference dose volume:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Compare dose volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ReferenceDoseVolume">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_CompareDoseVolume">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="toolTip">
         <string>Distance to agreement (DTA) tolerance, in mm</string>
        </property>
        <property name="text">
         <string>DTA distance tolerance (mm):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="toolTip">
         <string>Dose difference tolerance, defined as percentage of the reference dose (prescription dose) value</string>
        </property>
        <property name="text">
         <string>Dose difference tolerance (%):</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_8">
        <property name="toolTip">
         <string>Maximum gamma computed by the class. This is used to speed up computation. A typical value is 2 or 3</string>
        </property>
        <property name="text">
         <string>Maximum gamma:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_DtaDistanceTolerance">
        <property name="toolTip">
         <string>Distance to agreement (DTA) tolerance, in mm</string>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_DoseDifferenceTolerance">
        <property name="toolTip">
         <string>Dose difference tolerance. If a reference dose (prescription dose) is specified, the dose difference tolerance is treated as a percent of the reference dose. Otherwise it is treated as a percent of the maximum dose in the reference volume. To use a 3% dose tolerance, you would set this value to 0.03</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_MaximumGamma">
        <property name="toolTip">
         <string>Maximum gamma computed by the class. This is used to speed up computation. A typical value is 2 or 3</string>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_AnalysisThreshold">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Dose threshold for gamma analysis. This is used to ignore voxels which have dose below a certain value. For example, to consider only voxels which have dose greater than 10% of the prescription dose, set the analysis threshold to 10. The threshold is applied to the reference image&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_7">
        <property name="toolTip">
         <string>Dose threshold for gamma analysis. This is used to ignore voxels which have dose below a certain value. For example, to consider only voxels which have dose greater than 10% of the prescription dose, set the analysis threshold to 0.10. The threshold is applied to the reference image</string>
        </property>
        <property name="text">
         <string>Analysis threshold (%):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_Warning">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>128</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>128</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>106</red>
                <green>104</green>
                <blue>100</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>The selected volume is not a dose</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="toolTip">
         <string>Specify the 100% value for the dose difference tolerance</string>
        </property>
        <property name="text">
         <string>Reference dose:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_ReferenceDose_MaximumDose">
          <property name="toolTip">
           <string>Use the maximum value in the reference dose volume as reference dose</string>
          </property>
          <property name="text">
           <string>Use maximum dose</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_ReferenceDose_CustomValue">
          <property name="toolTip">
           <string>Use a user-defined value as reference dose</string>
          </property>
          <property name="text">
           <string>Use custom value (Gy):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_ReferenceDose">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Use a user-defined value as reference dose</string>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="toolTip">
         <string>Mask contour that defines the region that is considered for dose comparison</string>
        </property>
        <property name="text">
         <string>Mask contour:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_MaskContour">
        <property name="toolTip">
         <string>Mask contour that defines the region that is considered for dose comparison</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLContourNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_Output">
     <property name="text">
      <string>Output</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_GammaVolume">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>GammaVolume</string>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Gamma volume:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Pass fraction:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_PassFraction">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerDoseComparisonModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ParameterSet</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerDoseComparisonModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ReferenceDoseVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerDoseComparisonModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_CompareDoseVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerDoseComparisonModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_GammaVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerDoseComparisonModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_MaskContour</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>1</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
